docs/maintenance_or_infrequent_tasks.md

```diff
@@ -296,3 +296,35 @@ cf run-task crt-portal-django  -c "python crt_portal/manage.py refresh_trends" -
 Your local output of executing the above command will reflect success or failure of the task's submission.
 
 Output, if any, of the command being executed will be be available in the application logs.
+
+
+
+# Maintenance Mode
+
+We use an environment variable, `MAINTENANCE_MODE`, to run the application with altered functionality.
+
+To **enable** maintenance mode, we need to set the `MAINTENANCE_MODE` environment variable to `True` and restart all running instances.
+
+> **NOTE**: Cloud Foundry [CLI Version 7](https://docs.cloudfoundry.org/cf-cli/v7.html) is required to use `--strategy rolling`
+
+
+```shell
+cf target -s {target environment}
+cf set-env crt-portal-django MAINTENANCE_MODE True
+cf restart crt-portal-django --strategy rolling
+```
+
+To **disable** maintenance mode and return the application to normal operations, remove the `MAINTENANCE_MODE` variable from the desired environment and restart all running instances.
+
+```shell
+cf target -s {target environment}
+cf unset-env crt-portal-django MAINTENANCE_MODE
+cf restart crt-portal-django --strategy rolling
+```
```

Comment:
I think we should have a practice run with @ilodidoj on the above once we get this in. Love the documentation! :heart: